js/app.js

```diff
@@ -25,7 +23,26 @@ function MallItem(name,source){
     this.views =  0;
 //-----------------------------Add new "MallItem" to array------------------------------//
     MallItem.allItems.push(this);
-}
+//---------------------------------Local Storage-Lab 13--------------------------------//
+  }
+
+//This function translates array to JSON string
+  function updateStorage(){
```

Comment:
Nice work have a function that updates items from local storage and a different one to get items from storage.